packages/wdio-crossbrowsertesting-service/src/service.ts

```diff
@@ -4,7 +4,17 @@ import logger from '@wdio/logger'
 const log = logger('@wdio/crossbrowsertesting-service')
 const jobDataProperties = ['name', 'tags', 'public', 'build', 'extra']
 
-export default class CrossBrowserTestingService {
+export default class CrossBrowserTestingService implements WebdriverIO.ServiceInstance {
+    browser!: WebdriverIO.BrowserObject | WebdriverIO.MultiRemoteBrowserObject
+    testCnt: number;
+    failures: number;
```

Comment:
Default values can be defined here:

```suggestion
    private _testCnt = 0
    private _failures = 0
```

Reply:
I have to rewrite the entire test cases if we are keeping all the members in `service.ts` private. Currently all the test cases seems to have written by directly mutating the properties. Will give a shot though